ci-docker-images/oci-volume-provisioner-system-test/Makefile

```diff
@@ -12,17 +12,16 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-DOCKER_REGISTRY ?= iad.ocir.io
+OCIRREGISTRY ?= iad.ocir.io
+OCIRTENNANCY ?= oracle
```

Comment:
`OCIRTENANCY`  is this a typo?

Reply:
In the wercker pipeline the creds were named:

{code}
      username: $OCIRUSERNAME
      password: $OCIRPASSWORD
{code}

So this is ugly, but, consistent. Would have been nice if they all had an underscore, but, not really that important.

Reply:
You are right about tenancy though. Good spott!